CONTRIBUTING.md

```diff
@@ -1,55 +1,77 @@
 
-## Code Of Conduct
+## I don't want to read this whole thing I just have a question!!!
 
-This project adheres to the Adobe [code of conduct](CODE_OF_CONDUCT.md). By participating,
-you are expected to uphold this code. Please report unacceptable behavior to
-[Grp-opensourceoffice@adobe.com](mailto:Grp-opensourceoffice@adobe.com).
+> **Note:** [Please don't file an issue to ask a question.](https://github.com/adobe/spectrum-web-components/discussions) You'll get faster results by using the resources below.
 
-## Have A Question?
+We have an official message board with a detailed FAQ and where the community chimes in with helpful advice if you have questions.
 
-Start by filing an issue. The existing committers on this project work to reach
-consensus around project direction and issue solutions within issue threads
-(when appropriate).
+-   [Github Discussions, the official Spectrum Web Components message board](https://github.com/adobe/spectrum-web-components/discussions)
+-   [Spectrum Web Components FAQ](https://github.com/adobe/spectrum-web-components/discussions/categories/q-a)
 
-## Contributor License Agreement
+## How Can I Contribute?
+
+### Contributor License Agreement
 
 All third-party contributions to this project must be accompanied by a signed contributor
-license agreement. This gives Adobe permission to redistribute your contributions
-as part of the project. [Sign our CLA](http://opensource.adobe.com/cla.html). You
-only need to submit an Adobe CLA one time, so if you have submitted one previously,
-you are good to go!
+license agreement. [Sign our CLA](http://opensource.adobe.com/cla.html).
 
-## Code Reviews
+## Security Issues
 
-All submissions should come in the form of pull requests and need to be reviewed
-by project committers. Read [GitHub's pull request documentation](https://help.github.com/articles/about-pull-requests/)
-for more information on sending pull requests.
+Do not open up a GitHub issue if the bug is a security vulnerability in Rails, and instead to refer to our [security policy](https://helpx.adobe.com/security/alertus.html).
 
-Lastly, please follow the [pull request template](PULL_REQUEST_TEMPLATE.md) when
-submitting a pull request!
+## Did you find a bug?
 
-## Commitlint
+### Before Submitting A Bug Report
 
-We use [Commitlint](https://github.com/conventional-changelog/commitlint/#what-is-commitlint) to help manage the semantic versions across the various packages in this library. Please be sure that you take this into concideration when submitting PRs to this repositiory. Generally, your commits should look like the following:
+-   **Check the [Issue Discussions](https://github.com/adobe/spectrum-web-components/issues).** You might be able to find the cause of the problem and fix things yourself. Most importantly, check if you can reproduce the problem [in the latest version of Spectrum Web Components](https://studio.webcomponents.dev/workspace/adobe)
```

Comment:
```suggestion
-   **Check the [Issue Discussions](https://github.com/adobe/spectrum-web-components/issues).** You might be able to find the cause of the problem and fix things yourself. Most importantly, check if you can reproduce the problem [in the latest version of Spectrum Web Components](https://studio.webcomponents.dev/workspace/adobe). This will isolate the issue outside your particular use case and help us discuss the issue more directly.
```